### ISSUE TITLE
Restrict Primary Auth Group for Limited External Collaborators

### ISSUE THREAD

**Issue by torrin47** [USEPA/GeoPlatform-Request-Tool#135]
We recently noticed that many LEC users are also in a normal internal authgroup (OW, R5, etc), and we don't think this is intentional or appropriate, and we're pretty sure the source is the request tool. The whole concept of LEC is that these users don't get access to ANYTHING in the GPO that isn't explicitly shared with them. So they don't even have access to items shared with the entire Org. Putting LEC users in an EPA authgroup runs counter to that assumption because many authgroups have a whole bunch of items shared with them, and it's pretty clear they're not intended for LEC users. The dashboard will assume LEC users are associated with the primary authgroup of their sponsor for reporting purposes (and this can be adjusted later, if needed), so there's no need for the LEC users to also be a member of an internal authgroup.

TL:DR
In the Project Request form, if Geoplatform Role is set to Limited External Collaborator, force Primary Auth Group to be the Limited External Collaborator Auth Group with no other choices available:
![image](https://user-images.githubusercontent.com/2127036/230181632-c6b9bbe6-9193-4ef7-b1ba-201523957ab2.png)
If someone also includes an internal authgroup in the "GeoPlatform Groups" section above, that's ok.

**Comment by tbock** [comment]
@torrin47 We will transition to a configuration in the admin like so.  The choices available there are the ones currently selected as auth groups.  

![Image](https://github.com/USEPA/GeoPlatform-Request-Tool/assets/4040295/931cf59f-d48c-4119-80ae-d784cb236272)


This will in turn be used to limit choice in the frontend. 

To migrate existing setup is it correct to say EPA Publisher should have all current auth groups associated with it and LEC should ONLY have the LEC group.

**Comment by torrin47** [comment]
Yup, that's affirmative.  Sounds like a good approach.

**Comment by tbock** [comment]
How are auth groups handled in Geosecure? Is there such a thing? Just trying to determine if this needs to be required or optional based on environment or something. @torrin47

**Comment by torrin47** [comment]
No authgroups in Geosecure since most users are view-only. Less of a need to steward the users and their content - only the secure data.
